panoramapublic/src/org/labkey/panoramapublic/view/expannotations/experimentDetails.jsp

```diff
@@ -191,6 +212,19 @@
  {
      display: block;
  }
+ a.catalogIcon {
+     display: inline-block;
+     height: 20px;
+     width: 25px;
+ }
+ a.catalogIconGreen
+ {
+     background: url("../../PanoramaPublic/images/slideshow-icon-green.png") no-repeat bottom 0 right 0;
```

Comment:
Will this always be the right path? I worry about the `../../` part. Might be better to make it something like:

```suggestion
     background: url("<%= h(contextPath) %>/PanoramaPublic/images/slideshow-icon-green.png") no-repeat bottom 0 right 0;
```

Reply:
Done.